core/tests/recovery-test/src/index.ts

```diff
@@ -123,6 +123,10 @@ export enum NodeComponents {
     WITH_TREE_FETCHER_AND_NO_TREE = 'core,api,tree_fetcher'
 }
 
+export function withDAFetcher(components: NodeComponents): string {
+    return components.toString().concat(',da_fetcher');
```

Comment:
Nit: `components` is already a string (enums are syntactic sugar in TypeScript), so `toString()` is redundant. Also, `concat()` can be replaced with `+`.